src/Domain/MigrationStep/s160_ProductDraftMigration/ProductDraftImporter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Akeneo\PimMigration\Domain\MigrationStep\s160_ProductDraftMigration;
+
+use Akeneo\PimMigration\Domain\Command\ChainedConsole;
+use Akeneo\PimMigration\Domain\Command\MySqlExecuteCommand;
+use Akeneo\PimMigration\Domain\Command\MySqlQueryCommand;
+use Akeneo\PimMigration\Domain\Pim\DestinationPim;
+use Box\Spout\Common\Type;
+use Box\Spout\Reader\ReaderFactory;
+
+/**
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ */
+class ProductDraftImporter
+{
+    /** @var ChainedConsole */
+    private $console;
+
+    public function __construct(ChainedConsole $console)
+    {
+        $this->console = $console;
+    }
+
+    public function import(DestinationPim $pim, string $draftFilename)
+    {
+        $reader = ReaderFactory::create(Type::CSV);
+
+        $varDir = __DIR__ . '/../../../../var';
+        $reader->open($varDir .'/'. $draftFilename);
+
+        foreach ($reader->getSheetIterator() as $sheet) {
+            $hasHeader = true;
+            foreach ($sheet->getRowIterator() as $row) {
```

Comment:
I think there's a mistake. $hasHeader is set to false at the first iteration, so why doing a `foreach` ?